test/newNameLength.test.ts

```diff
@@ -0,0 +1,67 @@
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '../lib';
+import { kMaxLength } from 'buffer';
+import { AnyPrincipal } from 'aws-cdk-lib/aws-iam';
+import { BlueprintBuilder, stacks } from '../lib';
+import { z } from 'zod';
+import { Result } from 'aws-cdk-lib/aws-stepfunctions';
+
+const app = new cdk.App();
+
+const addOns: Array<blueprints.ClusterAddOn> = [
+  new blueprints.addons.MetricsServerAddOn(),
+  new blueprints.addons.ClusterAutoScalerAddOn(),
+  new blueprints.addons.VpcCniAddOn(),
+];
+
+//This makes an array result to hold pre made default blueprints without the build yet added for testing purposes.
+function getConstrainstDataSet(): BlueprintBuilder[] {
+  const result: BlueprintBuilder[] = [];
+  const blueprint1 = blueprints.EksBlueprint.builder().addOns(...addOns);
```

Comment:
please add specific data variations for each blueprint builder returned. They are identical atm.

Reply:
I have it that way for testing purposes and would remove duplicates when I am done. I just like having a few there in the situation I just needed one or two more to test.